.travis.yml

```diff
@@ -15,13 +15,13 @@ matrix:
       install:
         # CHANGED_FILES detects whether or not files related to the nightly build are changed.
         # True is so that the command returns a 0 exit code whether or not CHANGED_FILES is empty.
-        - CHANGED_FILES=$(git diff --name-only master..HEAD | grep "tests/examples\|examples") || true;
+        - CHANGED_FILES=$(git diff --name-only master..HEAD | grep "tests/examples\|examples|Dockerfile") || true;
         - if [[ "$TRAVIS_EVENT_TYPE" == "cron" || ! -z "$CHANGED_FILES" ]]; then
             INSTALL_LARGE_PYTHON_DEPS=true source ./travis/install-common-deps.sh;
           fi;
       script:
         - if [[ "$TRAVIS_EVENT_TYPE" == "cron" || ! -z "$CHANGED_FILES" ]]; then
-            pytest --verbose tests/examples --large;
+            pytest --verbose tests/examples --large && docker build -t mlflow . && docker image ls | grep mlflow;
```

Comment:
I would recommend separating the docker tests into its own check, i.e. check if `examples` is in `$CHANGED_FILES` to run the examples tests, and check if `Dockerfile` is in `$CHANGED_FILES` to run the new docker test. No need to run both tests if only one of the affected subsets of files are changed.

Reply:
I split the tests into two parts.  one for the `tests/examples` and the other for `Dockerfile`.  Note: I changed the test from checking to see if $CHANGED_FILES is empty string to testing for a substring, i.e,
```
if [[ "$TRAVIS_EVENT_TYPE" == "cron" ||  "$CHANGED_FILES" == *"Dockerfile"* ]]; then
```